### PR TITLE
Upgrading to Cordova 7.0.0

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

**Comment by brandonpage** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Tests have failed, see below for more information.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description</td>
    </tr>
  </tbody>
</table>

### Tests: 

Name | Classname | Time|
--- | --- | ---|
testUpsertConcurrentEntries | com.salesforce.androidsdk.phonegap.SmartStoreLoadJSTest | 1.234|
testSyncSObjectDetectConflictUpdate | com.salesforce.androidsdk.phonegap.SmartSyncJSTest | 0.302|


<p align="right" data-meta="generated_by_ci/circleci">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: libs/SalesforceHybrid/build.gradle [hunk 1]
```diff
@@ -15,7 +15,7 @@ apply plugin: 'com.jfrog.bintray'
 
 dependencies {
   api project(':libs:SmartSync')
-  api 'org.apache.cordova:framework:6.4.0:release@aar'
```
Comment (wmathurin) [inline libs/SalesforceHybrid/build.gradle]: What about the release@aar

Comment (bhariharan) [reply libs/SalesforceHybrid/build.gradle]: Looks like they uploaded it the right way this time, so we don't need it :P

Comment (bhariharan) [reply libs/SalesforceHybrid/build.gradle]: It'll fail with it.

**Comment by bhariharan** [comment]
Hybrid sample apps are working fine, tests are running too. Most of the changes in this version are around the generated project structure for `Android Studio 3.0`.

**Pull request merged**